src/purify.ts

```diff
@@ -9,32 +9,32 @@ export function purify(content: string) {
   const newContent = content
     /* wrap TS 2.2 enums w/ an IIFE */
     .replace(
-      /var (\S+) = \{\};\n(\1\.(\S+) = \d+;\n)+\1\[\1\.(\S+)\] = "\4";\n(\1\[\1\.(\S+)\] = "\S+";\n*)+/mg,
+      /var (\S+) = \{\};\r?\n(\1\.(\S+) = \d+;\r?\n)+\1\[\1\.(\S+)\] = "\4";\r?\n(\1\[\1\.(\S+)\] = "\S+";\r?\n*)+/mg,
       'var $1 = /*@__PURE__*/(function() {\n$&; return $1;})();\n',
     )
     /* wrap TS 2.3 enums w/ an IIFE */
     .replace(
       // tslint:disable-next-line:max-line-length
-      /var (\S+);(\/\*@__PURE__\*\/)*\n\(function \(\1\) \{\s+(\1\[\1\["(\S+)"\] = 0\] = "\4";(\s+\1\[\1\["\S+"\] = \d\] = "\S+";)*\n)\}\)\(\1 \|\| \(\1 = \{\}\)\);/mg,
+      /var (\S+);(\/\*@__PURE__\*\/)*\r?\n\(function \(\1\) \{\s+(\1\[\1\["(\S+)"\] = 0\] = "\4";(\s+\1\[\1\["\S+"\] = \d\] = "\S+";)*\r?\n)\}\)\(\1 \|\| \(\1 = \{\}\)\);/mg,
       'var $1 = /*@__PURE__*/(function() {\n    var $1 = {};\n    $3    return $1;\n})();',
     )
     /* Prefix safe imports with pure */
     .replace(
-      new RegExp(`(_(${pureImportMatches})__ = )(__webpack_require__\\("\\S+"\\));`, 'mg'),
```

Comment:
Regarding `"`, it sounds like perhaps you're not using the `HashedModuleIdsPlugin`in your tests. If you do use this plugin (as we have been in our manual testing), you get:

```ts
/* harmony import */ var __WEBPACK_IMPORTED_MODULE_2_rxjs_Subject__ = __webpack_require__("EEr4");
/* harmony import */ var __WEBPACK_IMPORTED_MODULE_2_rxjs_Subject___default = __webpack_require__.n(__WEBPACK_IMPORTED_MODULE_2_rxjs_Subject__);
```

It seems like we should support both cases: number or hash.